model/party.go

```diff
@@ -0,0 +1,10 @@
+package model
+
+type Party struct {
```

Comment:
I think we shouldn't start thinking about anything zone-server related. The project is still to immature for that. Instead, we should focus on account and char server related stuff.